scripts/generate_error_code.hs

```diff
@@ -96,8 +96,6 @@ invalidInputErrors = errorsEnumerate 100
     :? ("The sender has not been delegated the control of the required tokens.", NoArg)
   , "fail_decision_lambda"
     :? ("Executing the proposal's decision lambda results in failure.", NoArg)
-  , "entrypoint_not_found"
```

Comment:
Shouldn't `removedError` be used here instead?

Reply:
Yes, good catch. Changed.